template/deployments/helm/charts/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: {{APPNAME}}
+name: my-app
```

Comment:
I think this should be part of the gotemplate, this should be overwriteable. We can have this appname value be overwritten here and the created in the `Values.yaml` under appname to be referenced in other files